src/Frontend/src/components/audit/DatePickerRange.vue

```diff
@@ -43,7 +43,15 @@ function formatDate([fromDate, toDate]: DateRange) {
 </script>
 
 <template>
-  <VueDatePicker ref="datePicker" class="dropdown" v-model="internalModel" :format="formatDate" :range="{ partialRange: false }" :enable-seconds="true" :action-row="{ showNow: false, showCancel: false, showSelect: true }">
```

Comment:
The `:format` type, had issues, it seems the new version of the `vue-tsc` is more type safe and need an explicit cast.